index.js

```diff
@@ -55,24 +54,25 @@ app.get('/admin/login',function(req,res){
 
 app.post('/admin/login',function(req,res){
   var username = req.body.username;
-  var password = req.body.password;
-
-  db.get("SELECT EXISTS(SELECT * FROM superusers WHERE username='"+ username +"')",function(err,result){
-    console.log(result);
-    console.log(result[Object.keys(result)[0]])
-    if( result[Object.keys(result)[0]] == 1 ){
-      db.get("SELECT * FROM superusers WHERE username='"+ username +"'", (err, res1) => {
-        console.log(res1)
-        if( password == res1.password ){
-          req.session.name = "superuser"
-          res.redirect('/admin')
-        }else {
-          res.redirect('/')
-        }
-      })
-    }else {
-      res.redirect('/')
-    }
+  var hashed = crypto.createHmac('sha256',req.body.password).digest('hex');
```

Comment:
Crypto has a _key derivation_ API for hard-hashing stuff(~100000 times), it's better in this case.

https://nodejs.org/api/crypto.html#crypto_crypto_pbkdf2_password_salt_iterations_keylen_digest_callback